index.js

```diff
@@ -31,14 +35,14 @@ function format(data){
       out[k] = data[k];
     }
   });
-  return out; 
+  return out;
 }
 
-function findGit(dir, cb) {
-  var folder = path.join(dir, '.git/config')
+function findGit(dir,options,cb){
+  var folder = path.resolve(dir, process.env.GIT_DIR || options.gitDir || '.git', 'config');
```

Comment:
could we change this to have `options.gitDir` override `process.env.GIT_DIR`?
otherwise i could see this causing frustrating scenarios.